packages/vaadin-custom-field/test/visual/lumo/custom-field.test.js

```diff
@@ -1,5 +1,6 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
+import '@vaadin/combo-box/theme/lumo/vaadin-combo-box.js';
```

Comment:
Please add the new package to `devDependencies` in `vaadin-custom-field` package.json.